vumi/persist/riak_manager.py

```diff
@@ -293,16 +293,10 @@ def _search_iteration(self, bucket, query, rows, start):
 
     def real_search(self, modelcls, query, rows=None, start=None):
         rows = 1000 if rows is None else rows
+        start = 0 if start is None else start
```

Comment:
I'd prefer to throw an exception if we get `start=None` so we can find places that are using this incorrectly and fix them instead of having them silently return fewer results than they did before.